src/test/java/org/broadinstitute/hellbender/utils/io/BlockCompressIntervalStreamUnitTests.java

```diff
@@ -0,0 +1,178 @@
+package org.broadinstitute.hellbender.utils.io;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.SAMSequenceRecord;
+import htsjdk.samtools.seekablestream.ByteArraySeekableStream;
+import htsjdk.samtools.util.LocationAware;
+import htsjdk.tribble.*;
+import org.broadinstitute.hellbender.utils.codecs.FeaturesHeader;
+import org.broadinstitute.hellbender.utils.io.BlockCompressedIntervalStream.Reader;
+import org.broadinstitute.hellbender.utils.io.BlockCompressedIntervalStream.Writer;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class BlockCompressIntervalStreamUnitTests {
```

Comment:
Unit test classes should extend `GATKBaseTest`, and the class name should end in `*UnitTest`, not `*UnitTests` -- I believe this matters when we parallelize the test suite by test type in Travis.